### ISSUE TITLE
feat request: Preview Pro Icons in Neos Backend

### ISSUE THREAD

**Issue by paavo** [VIVOMEDIA/neos-fontawesome-icon#3]
Is there any way to preview to Pro Icons in Neos Backend?

**Comment by dlubitz** [comment]
No, unfortunately not. See: https://github.com/VIVOMEDIA/neos-fontawesome-icon#known-issues

There might by a way with the new Icon Feature, but this would need the SVG versions of the icons. 
https://www.neos.io/de/blog/neos-7-3-lts-released.html#neos-7-3-highlights

**Comment by paavo** [comment]
🤔  The SVGs (of the free icons) are all available in the [Fontawesome Github-Repository](https://github.com/FortAwesome/Font-Awesome/tree/master/svgs/brands). 
There is also a Pro-Repository, but this is not public. 

Maybe we can grab the Icons via npm.fontawesome.com?
